internal/envoy/auth/ext_authz_parser.go

```diff
@@ -33,7 +33,7 @@ import (
 	"github.com/kubeshop/kusk-gateway/pkg/options"
 )
 
-func ParseAuthUpstreamOptions(pathPrefix string, host options.AuthUpstreamHost, args *ParseAuthArguments, scheme string) error {
+func ParseAuthUpstreamOptions(pathPrefix string, host options.AuthUpstreamHost, args *ParseAuthArguments, scheme string, path *string) error {
```

Comment:
can't we have path property in AuthUpstreamHost?

Reply:
Also might be worth introducing an opts struct for parsing upstream options.
it is much cleaner than keep tacking on params on the function

Reply:
`path` property is now at `custom.host.path`.